react/finger-exercises/redux/02 - react-redux/src/app/screens/App/components/ShoppingCart/components/Item/index.js

```diff
@@ -1,19 +1,19 @@
 import React, { PureComponent } from 'react';
 import { func } from 'prop-types';
-import { bookSelectedPropType } from '@constants/propTypes';
 import Button from '@components/Button';
+import { bookSelectedPropType } from '@constants/propTypes';
 
 import styles from './styles.scss';
 
 class Item extends PureComponent {
-  addItem = () => {
-    const { item, addItem } = this.props;
-    addItem(item.id);
+  handleAddItem = () => {
+    const { item, onAddItem } = this.props;
+    onAddItem(item.id);
   };
```

Comment:
For less than 3 props it is not necessary to deconstruct `this.props`

Reply:
The result is `handleRemoveItem = () => this.props.onRemoveItem(this.item.id);`

"Me copa" 🤔